package.json

```diff
@@ -4,6 +4,9 @@
   "workspaces": [
     "packages/*"
   ],
+  "scripts": {
+    "build": "lerna run build"
```

Comment:
`lerna run build` at the root will effectively run `yarn build` in all the packages. Question for future consideration --- how do we want to build and publish the docs? I decided to not touch them for now. So currently, `yarn build` will run for the docs but I need to decided if we want to `.gitignore` the public directory, or if we somehow want to commit and publish the output of `yarn build` in the docs.

Caveat: Currently, if `yarn build` in swarm-docs produces a git diff, the build will error. You'll still need to run `yarn build` locally in the `swarm-docs` directory and commit the changes in order to not fail the build. This would also only happen if you're making changes to the docs.

Reply:
Yep. I'm pairing with @hunter right now and we're discussing this. Will file a follow up ticket.